uproot/write/objects/TH.py

```diff
@@ -270,7 +270,7 @@ def length_tattaxis(self):
     def return_taxis(self, cursor, axis):
         cnt = numpy.int64(self.length_taxis(axis) - 4) | uproot.const.kByteCountMask
         vers = 10
-        if axis["_fFirst"] != 0 or axis["_fLast"] != 0 or axis["_fBits2"] != 0 or axis["_fTimeDisplay"] or axis["_fTimeFormat"] != b"" or axis["_fLabels"] or axis["_fModLabs"]:
+        if axis["_fLabels"] or axis["_fModLabs"]:
```

Comment:
I prefer not to rely on Python's "truthiness" in boolean expressions. So many values can resolve to true or false that I'd much rather have something like

```python
if axis["_fLabels"] is not None or axis["_fModLabs"] is not None:
```

or even better for this case, since I know that only a minimum of fields should be required of these objects:

```python
if axis.get("_fLabels", None) is not None or axis.get("_fModLabs", None) is not None:
```

so that missing keys are treated the same way as `None` keys.

Reply:
I replaced the line with this - 
`if axis.get("_fLabels", None) != [] or axis.get("_fModLabs", None) != []:`